src/coreclr/jit/flowgraph.cpp

```diff
@@ -3354,6 +3354,34 @@ void Compiler::fgCreateFunclets()
 #endif // DEBUG
 }
 
+/*-------------------------------------------------------------------------
+ *
+ * Walk the EH funclet blocks of a function to determine if the funclet
+ * section is cold. If any of the funclets are hot, then it may not be
+ * beneficial to split at fgFirstFuncletBB, moving all funclets to the
+ * cold section.
+ */
```

Comment:
Can you use the "new style" function comments? They look like
```
//------------------------------------------------------------------------------
// methodName: short description
//
// Notes:
//   (text you have above)
//
bool Compiler::fgFuncletsAreCold()
```

Reply:
Sure thing; fixed.